setup_parser.bat

```diff
@@ -2,7 +2,9 @@
 rem See if python is already in the path
 set pypath=`where python`
```

Comment:
Seems like this is making Windows unhappy:

![Screen Shot 2020-08-14 at 1 12 47 PM](https://user-images.githubusercontent.com/51207431/90289070-f399e600-de2f-11ea-9ea4-e1c460b67ef8.png)

Reply:
reworked